cradle-core/src/main/java/com/exactpro/cradle/utils/TestEventUtils.java

```diff
@@ -80,23 +80,36 @@ public static void validateTestEvent(TestEvent event) throws CradleStorageExcept
 		if (event instanceof TestEventBatch && event.getParentId() == null)
 			throw new CradleStorageException("Batch must have a parent");
 		
-		if (event.getBookId() == null)
+		if (event.getBookId() == null || StringUtils.isEmpty(event.getBookId().toString()))
 			throw new CradleStorageException("Test event must have a book");
-		if (event.getScope() == null)
+		if (StringUtils.isEmpty(event.getScope()))
 			throw new CradleStorageException("Test event must have a scope");
 		if (event.getStartTimestamp() == null)
 			throw new CradleStorageException("Test event must have a start timestamp");
-		
-		if (event.getEndTimestamp() != null && event.getEndTimestamp().isBefore(event.getStartTimestamp()))
-			throw new CradleStorageException("Test event cannot end sooner than it started");
-		
+		Instant now = Instant.now();
+		if (event.getStartTimestamp().isAfter(now))
+			throw new CradleStorageException(
+					"Event start timestamp '" + TimeUtils.toLocalTimestamp(event.getStartTimestamp()) +
+							"' It can not be greater than current '" + TimeUtils.toLocalTimestamp(now) + "'");
```

Comment:
"Event start timestamp (" + TimeUtils.toLocalTimestamp(event.getStartTimestamp()) +
							") is greater than current timestamp (" + TimeUtils.toLocalTimestamp(now) + ")");